entrypoint.sh

```diff
@@ -274,6 +274,7 @@ zulipConfiguration() {
            [ "$setting_key" = "AUTH_LDAP_USER_ATTR_MAP" ] || \
            [ "$setting_key" = "AUTH_LDAP_USER_FLAGS_BY_GROUP" ] || \
            [ "$setting_key" = "AUTH_LDAP_GROUP_TYPE" ] || \
+           [ "$setting_key" = "SOCIAL_AUTH_OIDC_ENABLED_IDPS" ] || \
```

Comment:
It's technically a dict, not an array, but that's a pre-existing problem with entrypoint.sh. :(